internal-packages/graphql-test-server/src/schema.ts

```diff
@@ -1,4 +1,5 @@
 import { createSchema } from 'graphql-yoga';
+import { usePersistedOperations } from '@graphql-yoga/plugin-persisted-operations';
```

Comment:
this imported value seems unused? How is this passing linting?

Reply:
it's in `internal-packages` so maybe the linting is different? Either way, I'll remove it